pkg/sql/delegate/show_ranges.go

```diff
@@ -367,13 +369,29 @@ AND s.end_key > r.start_key`)
               ON r.table_id = t.table_id`, dbName.String(), dbNameCol)
 	}
 	buf.WriteString("\n),\n") // end of named_ranges CTE.
+	// If with details we need to get approximate_disk_bytes from tenant_span_stats.
+	if n.Options.Details {
+		buf.WriteString("span_stats AS (SELECT sum(approximate_disk_bytes) AS approx_disk_bytes FROM crdb_internal.tenant_span_stats(")
```

Comment:
Are you sure this is what we want? What if a range contains data from multiple tables? If I run `SHOW RANGE WITH TABLES` on a range containing multiple tables, i probably want to see the approx size for each of them.

Reply:
Ah thanks for pointing this out, I guess in that case we would not want to sum the rows.